docs/form-datasets.rst

```diff
@@ -1,32 +1,47 @@
+.. spelling:word-list::
+  aae
+  Ifedore
+  Magama
+  da
+  ee
+  Xue
+  Dia
+
 ************************
-Form Datasets
+Form datasets
 ************************
 
-ODK forms can use datasets in a variety of ways. These datasets can be either internal or external to the form.
+Workflows sometimes require access to historical data, standard data tables, lists of people, or other information. You can include that data directly in the form, attach it as a CSV file, or define it in an Entity List which can be shared between forms. All of these sources of data can be used to build select questions or to look up specific values.
 
 Internal datasets are defined in the **choices** sheet of an XLSForm and are typically used as choices for :ref:`selects <select-widgets>`. You can also define a dataset in the **choices** sheet to :ref:`look up values <referencing-values-in-datasets>` based on user input. Learn how to define internal datasets in the section on :ref:`selects <select-widgets>`.
 
-External datasets are useful when:
+External datasets can either be files that get attached to the form or :doc:`Entity Lists <central-entities>`. Entity Lists can be thought of as CSVs that are managed by ODK Central so that data collected by one form submission can be automatically accessible in other forms in the same project. The also can be shared between forms and can be updated by form submissions.
```

Comment:
The also -> They also

The they is a little vague, so you can say "Entity Lists can also be"